add docs for redeliver_event api docs (fix #4176) (#4177)
hasura/graphql-engine: docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst

     - TriggerName_
     - Name of the event trigger

.. _redeliver_event:

redeliver_event
---------------

> ``redeliver_event`` is used to redeliver an existing event. For e.g., if an event is marked as error (

Reviewer: ```suggestion
``redeliver_event`` is used to redeliver an existing event. For example, if an event is marked as error (
```

> say it did not succeed event after retries) then you can redeliver it using this API. Note that this will reset the count of retries so far,

Reviewer: ```suggestion
say it did not succeed after retries), you can redeliver it using this API. Note that this will reset the count of retries so far.
```

> if the event fails to deliver then it will be retried automatically according to it's ``retry_conf``.

Reviewer: ```suggestion
If the event fails to deliver, it will be retried automatically according to its ``retry_conf``.
```
